examples/source/1.components/amp-date-picker.html

```diff
@@ -85,6 +85,15 @@
     margin-top: var(--space-1);
     width: 27px;
   }
+  
+  /* edit inline-code-preview */
```

Comment:
Some more details would help 😉What behaviour does that rule prevent specifically - basically try to explain what you would have liked to know when you were starting with this problem.